wallet-new/server/Cardano/Wallet/LegacyServer.hs

```diff
@@ -4,7 +4,8 @@ module Cardano.Wallet.LegacyServer where
 import           Cardano.Wallet.API
 import           Cardano.Wallet.API.V1.Migration as Migration
 
-import qualified Cardano.Wallet.API.V0.Handlers       as V0
+import qualified Cardano.Wallet.API.V0.Handlers as V0
+import qualified Cardano.Wallet.API.Dev.LegacyHandlers as Dev
```

Comment:
What's wrong with `Development` instead ? I like long (well, reasonably long 😁 ) and explicit names.

Reply:
So that would be `import qualified Cardano.Wallet.API.Development.LegacyHandlers as Development`

Reply:
Oh maaaan, what's wrong with `Dev` instead of `Development`? `Dev` is self-explaining, too. And we do use short modules names everywhere (e.g. `V0` instead of `Version0`). Anyway, I don't care about names and will change it if you really want.

Reply:
>  Anyway, I don't care about names and will change it if you really want.

Not really a dictator, just thinking out loud here 😁  Maybe the middle ground would be to at least name the module (that you import once) `Development` but use `Dev` to qualify the imports (which is something done frequently)?